src/commands/lambda/renderers/flare-renderer.ts

```diff
@@ -1,8 +1,8 @@
-import {dryRunTag} from './common-renderer'
+import {dryRunTag, errorTag, warningExclamationSignTag} from './common-renderer'
 
 /**
  * @returns a header indicating which `lambda` subcommand is running.
- * @param command current selected lambda subcommand.
+ * @param isDryRun whether or not the command is a dry run. Defaults to false.
```

Comment:
This is not true, since the function signature is not `= (isDryRun = false)`. So there is no default.